src/session_impl.cpp

```diff
@@ -170,20 +170,7 @@ namespace boost {
 }
 #endif
 
-namespace libtorrent {
-
-#if defined TORRENT_ASIO_DEBUGGING
-	std::map<std::string, async_t> _async_ops;
-	std::deque<wakeup_t> _wakeups;
-	int _async_ops_nthreads = 0;
-	std::mutex _async_ops_mutex;
-
-	std::map<int, handler_alloc_t> _handler_storage;
-	std::mutex _handler_storage_mutex;
-	bool _handler_logger_registered = false;
-#endif
```

Comment:
what happened here?

Reply:
what I mean is; this should not be removed

Reply:
Looks like clang-tidy goofed. I'll manually fix.